kademlia/src/main/java/ingsw/group1/kademlia/KadActionsBuilder.java

```diff
@@ -38,17 +36,17 @@ public KadActionsBuilder(int maxActionID) {
      * Default constructor, builder will accept all valid IDs, (as indicated in KadAction)
      */
     public KadActionsBuilder() {
-        MAX_ACTION_ID = KadAction.MAX_ID;
+        maxActionId = KadAction.MAX_ID;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4761148)